README.md

```diff
@@ -100,18 +69,64 @@ Much of the philosophy behind Bedrock is inspired by the [Twelve-Factor App](htt
 - `WP_SITEURL` - Full URL to WordPress including subdirectory (https://example.com/wp)
 - `AUTH_KEY`, `SECURE_AUTH_KEY`, `LOGGED_IN_KEY`, `NONCE_KEY`, `AUTH_SALT`, `SECURE_AUTH_SALT`, `LOGGED_IN_SALT`, `NONCE_SALT`
   - Generate with [wp-cli-dotenv-command](https://github.com/aaemnnosttv/wp-cli-dotenv-command)
-  - Generate with [our WordPress salts generator](https://roots.io/salts.html)
+  - Regenerate with [Bedrock's WordPress salts generator](https://roots.io/salts.html)
+
+### WordPress Config
+
+The `wp-config.php` file is located in the `web` directory. As with other WordPress sites on Pantheon, much of this is taken care of for you in `wp-config-pantheon.php`. Application-level configuration takes place in `config/application.php`. This can be referenced as a guide to understand how the constants are set up and how the `.env` files work, but modifying this file may result in merge conflicts and is not recommended. Any configuration changes should be made to your `wp-config.php` file directly.
+
+You can learn more about WordPress configuration with Bedrock in the [Bedrock Configuration docs](https://docs.roots.io/bedrock/master/configuration/).
+
+### Understanding the WordPress codebase
+
+Bedrock installs WordPress as a required package so updates can be managed by Composer. As such, the contents of the `wp-content` directory have been moved outside the WordPress codebase so changes can be made safely to files within those directories without conflicts.
+
+* Theme are installed into `web/app/themes/`.
+* Plugins are installed into `web/app/plugins`.
+* Must-use plugins are installed into `web/app/mu-plugins`.
+* The WordPress admin dashboard is available at `https://example.com/wp/wp-admin/`
+
+### Using Composer to manage plugins and themes
+
+[Packagist](https://packagist.org) is a repository of Composer packages that are available by default to projects managed by Composer. Packagist libraries receive updates from their source GitHub repositories automatically.
+
+[WPackagist](https://wpackagist.org) is a Packagist-like mirror of the WordPress.org [plugin](https://wordpress.org/plugins) and [theme](https://wordpress.org/themes) repositories and is included with Bedrock out of the box. 
+
+You may install packages from Packagist or WPackagist without any additional configuration using `composer require`.
+
+#### Requiring a package from Packagist
+
+Some WordPress developers push their packages to Packagist in addition to the WordPress plugin and theme repositories. In this way, it may be beneficial to pull those packages directly from Packagist to get the latest code directly from the source.
+
+```
+composer require yoast/wordpress-seo
```

Comment:
🔢 
```suggestion
`composer upstream-require yoast/wordpress-seo`
```

Change the few instances of these commands so that users don't modify the root composer.json file.